### PR TITLE
Ppt or g per kg

### PR DESCRIPTION
**Pull request by jbathmann**
Suggestion to maintain consistency
This PR contains
- ppt to g/kg in simple hydro
- updated example project
- updates in doc
an is related to issue #81

### CODE REVIEW

**Review by mcwimm — COMMENTED** [review]
I think you mixed up the units gain. If we want SI, it's kg/kg and not g/kg aka ppt

File: TreeModelLib/BelowgroundCompetition/SimpleHydro/SimpleHydro.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def __init__(self, args):
     def calculateBelowgroundResources(self):
         self.transpire()
         self.belowground_resources = ((np.array(self._potential_nosal) +
-                                       np.array(self._salinity) * 85000) /
+                                       np.array(self._salinity) * 85000000) /
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply TreeModelLib/BelowgroundCompetition/SimpleHydro/SimpleHydro.py:29]: right. I changed everything to kg/kg, Sorry!

File: TreeModelLib/BelowgroundCompetition/SimpleHydro/SimpleHydro.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def transpire(self):
                                    np.sum(presence[:, :, ii])))
             ### transpiration for the tree
             self.transpiration.append(
-                (-self._potential_nosal[ii] - self._salinity[ii] * 85000) /
+                (-self._potential_nosal[ii] - self._salinity[ii] * 85000000) /
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply TreeModelLib/BelowgroundCompetition/SimpleHydro/SimpleHydro.py:70]: ..

File: test/SmallTests/Test_Setups_small/SimpleHydro_SAZOI_BETTINA.xml [hunk 3]
```diff
@@ -19,9 +19,9 @@
             <dilution_frac_upper> 0.0 </dilution_frac_upper>
             <dilution_frac_lower> 0.2 </dilution_frac_lower>
             <diffusion_frac> 0.05 </diffusion_frac>
-            <sea_salinity> 35 </sea_salinity>
-            <ini_sal> 45 </ini_sal>
-            <up_sal> 10 </up_sal>
+            <sea_salinity> 0.035 </sea_salinity>
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply test/SmallTests/Test_Setups_small/SimpleHydro_SAZOI_BETTINA.xml:22]: ..

File: Documentation/ProjectFile/MangaProject/tree_dynamics/belowground_competition/SimpleHydro/s_ini_sal.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 
 
-initial salinity in the soil in ppt.
+initial salinity in the soil in g/kg.
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply Documentation/ProjectFile/MangaProject/tree_dynamics/belowground_competition/SimpleHydro/s_ini_sal.md]: changed to kg/kg

**Comment by mcwimm** [comment]
Thanks for updating all the files.

**Pull request merged**